pinot-tools/src/main/resources/examples/batch/githubEvents/githubEvents_offline_complexTypeHandling_table_config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "tableName": "githubEvents",
+  "tableType": "OFFLINE",
+  "tenants": {
+  },
+  "segmentsConfig": {
+    "segmentPushType": "REFRESH",
+    "replication": "1",
+    "timeColumnName": "created_at_timestamp"
+  },
+  "tableIndexConfig": {
+    "loadMode": "MMAP"
+  },
+  "ingestionConfig": {
+    "transformConfigs": [
+      {
+        "columnName": "created_at_timestamp",
+        "transformFunction": "fromDateTime(created_at, 'yyyy-MM-dd''T''HH:mm:ss''Z''')"
+      },
+      {
+        "columnName": "payload_json",
+        "transformFunction": "jsonFormat(\"payload\")"
+      }
+    ],
+    "complexTypeConfig": {
+      "unnestFields": ["payload.commits"]
+    }
```

Comment:
Can we please add another quickstart example where the entire payload json column is unnested as opposed to payload.commits subtree ? Someone interested in using nested data this way without json index will find it helpful

Reply:
I don't get your question. Any collections to unnest shall be configured. If you refer to `flatten maps`, then this quickstart example already does it.

Reply:
I understand that any collections to unnest can be configured. Assuming payload is the name of a json/nested column, I was thinking of adding one example demonstrating the complete flattening from the root level

Reply:
Discussed offline, this would be easier to add after the next PR on schema inference. so we dont have to list out all the columns manually